### PR TITLE
7.0 pep8

### PR DESCRIPTION
**Pull request by bwrsandman**
First part of pep8 fixes on repo

Depends on ~~#28~~

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Travis doesn't inform about PEP8 issues in the modules you mention, but it will be better to complete PEP8 fix in the same PR. If this a work in progress, please mark it with the corresponding label.

**Comment by bwrsandman** [comment]
This PR is ready and doesn't aim to fix unittests, only pep8

**Comment by pedrobaeza** [comment]
PEP8 OK, so :+1:

File: account_banking/__init__.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 #    garantees and support are strongly adviced to contract EduSense BV
 #
 #    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as published by
+#    it under the terms of the GNU Affero General Public License as published
```
Comment (CSester) [inline account_banking/__init__.py:14]: Missing 'by' here.
Everything else seems good! :+1:

**Comment by pedrobaeza** [comment]
Please rebase and let's try to speep up this PR

**Comment by pedrobaeza** [comment]
Two little PEP8 errors:

```
./account_banking_partner_journal_account/res_partner.py:67:80: E501 line too long (80 > 79 characters)
./account_banking_camt/__openerp__.py:7:80: E501 line too long (80 > 79 characters)
```

**Comment by bwrsandman** [comment]
@pedrobaeza Done

**Comment by pedrobaeza** [comment]
OK, PEP8 is fixed. It is still red, but because of another issue. If you are OK, I'm going to merge this, because I need for the base of the 8.0, that I have ready a big migration PR.

:+1:

**Comment by guewen** [comment]
:+1:

**Pull request merged**